src/interval.jl

```diff
@@ -222,11 +222,11 @@ end
 
 # random sampling from interval
 Random.gentype(::Type{Interval{L,R,T}}) where {L,R,T} = float(T)
-function Random.rand(rng::AbstractRNG, i::Random.SamplerTrivial{ClosedInterval{T}}) where {T<:Real}
+function Random.rand(rng::AbstractRNG, i::Random.SamplerTrivial{<:TypedEndpointsInterval{:closed, R, T}}) where {R, T<:Real}
```

Comment:
`clamp` doesn't support `TypedEndpointsInterval{:closed,:open}`, so the type parameter `R` should be replaced with `:closed`. See https://github.com/JuliaMath/IntervalSets.jl/pull/104/files#r846726630 for the reason for the `clamp`.

Reply:
Done!